internal/api/ui/login/static/templates/ldap_login.html

```diff
@@ -0,0 +1,35 @@
+{{template "main-top" .}}
+
+<div class="lgn-head">
+    <h1>{{t "Login.Title"}}</h1>
+    <p>{{t "Login.Description"}}</p>
+</div>
+
+
+<form action="{{ ldapUrl }}" method="POST">
+
+    {{ .CSRF }}
+
+    <input type="hidden" name="authRequestID" value="{{ .AuthReqID }}"/>
+
+    <div class="fields">
+        <label class="lgn-label" for="username">{{t "Login.LoginNameLabel"}}</label>
+        <input class="lgn-input lgn-suffix-input" type="text" id="username" name="username" autocomplete="username" autofocus required>
+        <input class="lgn-input lgn-suffix-input" type="password" id="password" name="password" autocomplete="current-password" autofocus required>
+    </div>
```

Comment:
without custom texts or at least specific texts this might be confusing:
and autocomplete will interfere with the zitadel credentials
![image](https://user-images.githubusercontent.com/9405495/222067114-5794c25c-051a-4f20-9db0-440284cdf468.png)